### ISSUE TITLE
Incorrect exercise: [courseId=grammar-intro_eng][lessonNumber=3][exerciseId=2431]

### ISSUE THREAD

**Issue by sisima70** [jqueiroz/lojban.io#170]
# Description
Idiomatically, "People like us" means <lo prenu ku nelci *mi'o*>, so it's a bad example.

# Diagnostics information (do not remove)
- Exercise identifier: [courseId=grammar-intro_eng][lessonNumber=3][exerciseId=2431]
- Exercise payload: {"shouldDisplayHint":false,"exercise":{"canonicalAnswer":"lo prenu ku nelci mi","sentences":[{"text":"People like us.","lojbanic":false}],"title":"Translate this sentence","type":"typing"}}
- Attempted solution: {"text":"lo prenu ku nelci mi"}
- Attempted solution was classified as correct: yes
- Validation payload: {"canonicalAnswer":"lo prenu ku nelci mi","correct":true}

**Comment by vinivosh** [comment]
True. Maybe a clarification for the sentence would make this unambiguous? Or mabe even `There exists people that like us` instead?

**Comment by sisima70** [comment]
Returning to this issue, I now think "People like us." could mean either <lo prenu ku nelci mi> or <lo ... mi'o>. Do we add more answers that are considered valid (might require a change in the core engine to be able to, but if I remember and can find the time I could probably do that myself), or do we add another sentence to the question text?
